flagger.py

```diff
@@ -146,33 +146,26 @@ def message_destination(self, message):
         reactions = message.get("reactions")
         emoji_set = set(self.emoji)
         current_reactions = {}
-        t = message.get("text")
-        if t.find("SVP") != -1:
```

Comment:
Nope, both branches of this were the same, and defining a noop inner function, calls to which I changed to logger.debug calls.